ltp/data.py

```diff
@@ -124,16 +163,47 @@ def read_runtest(suite_name: str, content: str) -> Suite:
         if len(parts) >= 3:
             test_args = parts[2:]
 
-        test = Test(test_name, test_cmd, test_args)
+        parallelizable = True
+
+        if not metadata_tests:
+            # no metadata no party
+            parallelizable = False
+        else:
+            test_params = metadata_tests.get(test_name, None)
+            if test_params:
+                LOGGER.info("Found %s test params in metadata", test_name)
+                LOGGER.debug("params=%s", test_params)
+
+            if test_params is None:
+                # this probably means test is not using new LTP API,
+                # so we can't decide if test can run in parallel or not
+                parallelizable = False
+            else:
+                for blacklist_param in PARALLEL_BLACKLIST:
+                    if blacklist_param in test_params:
+                        parallelizable = False
+                        break
+
+        if not parallelizable:
+            LOGGER.info("Test '%s' is not parallelizable", test_name)
+        else:
+            LOGGER.info("Test '%s' is parallelizable", test_name)
+
+        test = Test(
+            test_name,
+            test_cmd,
+            test_args,
+            parallelizable=parallelizable)
```

Comment:
I guess that we will eventually make use of more test metadata, so maybe we should store the whole test_params into the test object, but I guess that we can do that later on.